juneau-core/juneau-marshall/src/main/java/org/apache/juneau/internal/IOUtils.java

```diff
@@ -473,10 +474,7 @@ public static int getBufferSize(String contentLength) {
 	 * @param is The input stream to close.
 	 */
 	public static void closeQuietly(InputStream is) {
-		try {
-			if (is != null)
-				is.close();
-		} catch (IOException e) {}
+

```

Comment:
This code shouldn't be removed.  The IOUtils.closeQuietly(...) methods are general utility methods for closing streams.

Reply:
This method calls is.close() and is is an InputStream. Java's documentation states that [InputStream close()](https://docs.oracle.com/javase/8/docs/api/java/io/InputStream.html#close--) doesn't do anything.

Since is.close() doesn't do anything (including throwing an IOException), having a closeQuietly method is IMHO a little confusing.

Reply:
The Java documentation is misleading.  The close() method on the InputStream class itself may be a no-op, but it is very much not a no-op on subclasses of InputStream (FileInputStream for example).

Reply:
Done.